common/templates/general.go

```diff
@@ -240,6 +241,29 @@ func CreateMessageSend(values ...interface{}) (*discordgo.MessageSend, error) {
 				ContentType: "text/plain",
 				Reader:      &buf,
 			}
+
+		case "image":
+			if val == nil {
+				return nil, errors.New("need an arguement for the image")
```

Comment:
```suggestion
				return nil, errors.New("need an argument for the image")
```
Small typo.

Side note: I don't really feel that making YAGPDB fetch potentially very large user-provided images is a great idea, but obviously the decision regarding whether or not this PR will be accepted isn't up to me : )